shell/platform/windows/fixtures/main.dart

```diff
@@ -159,6 +159,26 @@ void enableLifecycleToFrom() async {
   });
 }
 
+@pragma('vm:entry-point')
+void sendCreationMethod() async {
+  final List<int> data = <int>[
+    // Method name
+    7, 'create'.length, ...utf8.encode('create'),
+    // Method arguments: {'type': 'type':, 'id': 0}
+    13, 2,
+    7, 'type'.length, ...utf8.encode('type'),
+    7, 'type'.length, ...utf8.encode('type'),
+    7, 'id'.length, ...utf8.encode('id'),
+    3, 0, 0, 0, 0,
+  ];
+
+  final Completer<ByteData?> completed = Completer<ByteData?>();
```

Comment:
Question - what is `completed` being used for? It doesn't look like an associated `Future` for this is ever used.

Reply:
I intended to await its future in the fixture. I will add this.